charts/mlflow/templates/deployment.yaml

```diff
@@ -66,22 +66,26 @@ spec:
             - name: http
               containerPort: 5000
               protocol: TCP
+          {{- if .Values.livenessProbe.enabled }}
           livenessProbe:
             httpGet:
-            {{- if .Values.extraArgs.staticPrefix }}
-              path: {{ .Values.extraArgs.staticPrefix }}
-            {{- else }}
-              path: /
-            {{- end }}
+              path: {{ .Values.livenessProbe.healthCheckPath default "/" }}
```

Comment:
I didn't understand why people need to set it additionally. If they need a static prefix for their's endpoint, they will set it to a static prefix extra argument. Otherwise, it will be always slash.